src/Driver/Driver.jl

```diff
@@ -100,14 +101,14 @@ function parse_commandline()
         "--diagnostics-interval"
             help = "interval in simulation steps for gathering diagnostics"
             arg_type = Int
-            default = 10000
+            default = 1
```

Comment:
I don't think it's a good idea to make this the default. :)

Reply:
🥇